### ISSUE TITLE
Req: Add MS Windows support for new Makefile

### ISSUE THREAD

**Issue by btoschek** [btoschek/hsc-template#11]
Due to myself being a full-time Linux user, I can't implement Windows-specific Makefile functionality. There's about 3 commands that need Windows-specific implementations (and infrastructure to support them).
This could be implemented similar to the current version:

```make
# Windows commands
ifeq ($(OS),Windows_NT)
  COMMAND_1 := ...

# Unix-like commands
else
  COMMAND_1 := ...
endif
```

A working version of the new Makefile can be found at [hsc-template/makefile-rework](https://github.com/btoschek/hsc-template/tree/makefile-rework). This still needs a bit of fine-tuning (some unnecessary compile steps when index files haven't changed), but should for the most part improve compilation experience drastically.

**Comment by HannesLueer** [comment]
Hi, I have created a pull request (#14) for this issue. Maybe someone can take a look at it?

**Comment by btoschek** [comment]
Looks good to me. I'll have someone run the build process on their machine, just to be 100% sure. Will get back to you when finished. Thanks 👍.

**Comment by btoschek** [comment]
Changes merged in 7376a69, closing.
